pkg/instancetype/instancetype_test.go

```diff
@@ -1373,12 +1380,82 @@ var _ = Describe("Instancetype and Preferences", func() {
 					},
 				}
 
-				conflicts := instancetypeMethods.ApplyToVmi(field, instancetypeSpec, preferenceSpec, &vmi.Spec)
+				conflicts := instancetypeMethods.ApplyToVmi(field, instancetypeSpec, preferenceSpec, &vmi.Spec, &vmi.ObjectMeta)
 				Expect(conflicts).To(HaveLen(1))
 				Expect(conflicts[0].String()).To(Equal("spec.template.spec.domain.devices.hostDevices"))
 			})
 		})
 
+		Context("Instancetype.Spec.Annotations", func() {
+
+			BeforeEach(func() {
+				instancetypeSpec = &instancetypev1beta1.VirtualMachineInstancetypeSpec{
+					Annotations: make(map[string]string),
+				}
+			})
+
+			It("should apply to VMI", func() {
+				instancetypeSpec.Annotations = multipleAnnotations
+
+				conflicts := instancetypeMethods.ApplyToVmi(field, instancetypeSpec, nil, &vmi.Spec, &vmi.ObjectMeta)
+				Expect(conflicts).To(BeEmpty())
+				Expect(vmi.Annotations).To(Equal(instancetypeSpec.Annotations))
+			})
+
+			It("should not detect conflict when annotation with the same value already exists", func() {
+				instancetypeSpec.Annotations = multipleAnnotations
+				vmi.Annotations = multipleAnnotations
+
+				conflicts := instancetypeMethods.ApplyToVmi(field, instancetypeSpec, nil, &vmi.Spec, &vmi.ObjectMeta)
+				Expect(conflicts).To(BeEmpty())
+				Expect(vmi.Annotations).To(Equal(instancetypeSpec.Annotations))
+			})
+
+			It("should detect conflict when annotation with different value already exists", func() {
+				instancetypeSpec.Annotations = singleAnnotation
```

Comment:
I'm not sure that we really need `singleAnnotation` for this one test, maybe use `multipleAnnotations` and move the assignment into the `BeforeEach`?